app/views/jobseekers/job_applications/show.html.slim

```diff
@@ -0,0 +1,38 @@
+- content_for :page_title_prefix, t(".title")
+
+.jobs-banner
+  .govuk-width-container
+    .govuk-grid-row
+      .govuk-grid-column-full
+        = render(Shared::BreadcrumbComponent.new(collapse_on_mobile: false,
+                                                 crumbs: [{ link_path: jobseekers_job_applications_path, link_text: t("breadcrumbs.job_applications") },
+                                                          { link_path: "#", link_text: job_application.vacancy.job_title }]))
+
+        .govuk-caption-l class="govuk-!-margin-top-5" = t("jobseekers.job_applications.heading_component.heading")
+
+        h2.govuk-heading-xl class="govuk-!-margin-bottom-5" = vacancy.job_title
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1.govuk-heading-l = t(".title")
```

Comment:
In https://github.com/DFE-Digital/teaching-vacancies/pull/2985 I suggested that we start to distinguish between the `title` and `heading` text, so that it can differ.

Reply:
But maybe it's OK here as the texts don't necessarily need to differ. Content Design has not provided a title in https://docs.google.com/document/d/1oF4Vh1P8eJRdzmlJ4--cDDZQ_bCZMkKv/edit#heading=h.gjdgxs

Reply:
I'll support it anyway